os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -920,7 +921,7 @@ static void _up_assert(int errorcode)
 {
 	/* Are we in an interrupt handler or the idle task? */
 
-	if (g_upassert || current_regs || (this_task())->pid == 0) {
+	if (current_regs || (this_task())->pid == 0) {
```

Comment:
Any change done as part of this PR needs to be under FAULT_MGR config flag. Let us not change anything directly in the base code.

Reply:
Modified the code accordingly